### PR TITLE
test case to move cards to anohter board

### PR DESCRIPTION
**Pull request by nicolasvallejo13**
…r board

´´´
Verify A Card Can Be Created                                          | PASS |
------------------------------------------------------------------------------
Modify A Card's Name From The Board Page                              | PASS |
------------------------------------------------------------------------------
Leave A Comment On A Card                                             | PASS |
------------------------------------------------------------------------------
Verify A Card Can Be Deleted                                          | PASS |
------------------------------------------------------------------------------
Card Can Be Dragged To Another List                                   | PASS |
------------------------------------------------------------------------------
Card Can Be Moved To Another Board                                    ...[8804:15044:0624/171656.288:ERROR:device_event_log_impl.cc(214)] [17:16:56.287] USB: usb_device_handle_win.cc:1048 Failed to read descriptor from node connection: Uno de los dispositivos conectados al sistema no funciona. (0x1F)
[8804:15044:0624/171656.296:ERROR:device_event_log_impl.cc(214)] [17:16:56.295] USB: usb_device_handle_win.cc:1048 Failed to read descriptor from node connection: Uno de los dispositivos conectados al sistema no funciona. (0x1F)
Card Can Be Moved To Another Board                                    | PASS |
------------------------------------------------------------------------------
Trello Gui Cards :: Test Cases for Cards Feature                      | PASS |
6 tests, 6 passed, 0 failed
´´´

### CODE REVIEW

File: tests/trello/gui/resources/components/TrelloUpperMenu.resource [hunk 1]
```diff
@@ -0,0 +1,12 @@
+*** Settings ***
+Documentation    Variables and Keywords for Trello's Upper menu
+...              (can be seen from all over the web app)
+
+
+*** Variables ***
+### Locators ###
+${TRELO_ICON}             xpath://a[@aria-label='Back to home']
```
Comment (mefemefe) [inline tests/trello/gui/resources/components/TrelloUpperMenu.resource:8]:
```suggestion
${TRELLO_ICON}             xpath://a[@aria-label='Back to home']
```

File: tests/trello/gui/resources/components/TrelloUpperMenu.resource [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+*** Keywords ***
+Go Home
+     Click Element           ${TRELO_ICON}
```
Comment (mefemefe) [inline tests/trello/gui/resources/components/TrelloUpperMenu.resource:12]:
```suggestion
     Click Element           ${TRELLO_ICON}
```

File: tests/trello/gui/resources/components/CardModal.resource [hunk 3]
```diff
@@ -45,3 +48,15 @@ Get Number Of Comments
...
+    [Documentation]    Moves the card using the "move" button in Actions
+    ...                And Selects the board N° above 
+    ...                required: number (number of boards that the card will be moved above)
+    [Arguments]        ${number}
+    Wait Until Element Is Visible    ${MOVE_BUTTON} 
+    Click Element    ${MOVE_BUTTON}
+    Click Element    ${SELECT_DESTINATION}
+    Repeat Keyword    ${number}         Press Keys   ${SELECT_DESTINATION}         \uE013
+    Press Keys   ${SELECT_DESTINATION}         \uE007
+    Click Element    ${ACEPT_MOVE_BUTTON}
```
Comment (mefemefe) [inline tests/trello/gui/resources/components/CardModal.resource:62]:
```suggestion
    Click Element    ${ACCEPT_MOVE_BUTTON}
```

File: tests/trello/gui/trello_gui_cards.robot [hunk 4]
```diff
@@ -65,6 +67,21 @@ Card Can Be Dragged To Another List
...
+    Go To Board       CardTestBoard2
+    Set Test Variable         \${list}    To Do
+    Set Test Variable         \${card}    Test Card2
+    Create Card In List        ${list}    ${card}
+    Go To Card Modal           ${list}    ${card}
+    Move Card To An Above Board     1
+    ${Cards_On_list}=       Get Number Of Cards In List       To Do
+    ${0}=	                 Convert To Integer	             0
+    Should Be Equal             ${Cards_On_list}        ${0}
+    Wait Until Keyword Succeeds    10    2.5    Delete A Board            ${board2.json()}[id]
```
Comment (mefemefe) [inline tests/trello/gui/trello_gui_cards.robot:83]: The process of deleting the board should be a teardown

Comment (mefemefe) [inline tests/trello/gui/trello_gui_cards.robot:83]: The process of deleting the board should be a teardown

**Review by mefemefe — CHANGES_REQUESTED** [review]

**Review by sebasonti-fundjala — APPROVED** [review]

File: tests/trello/gui/trello_gui_cards.robot [hunk 5]
```diff
@@ -4,21 +4,26 @@ Resource          resources/LoginPage.resource
 Resource          tests/trello/api/resources/Board.resource
 Resource          resources/BoardsPage.resource
 Resource          resources/BoardPage.resource
+Resource          resources/components/TrelloUpperMenu.resource
 Suite Setup       Cards Suite Setup
 Suite Teardown    Cards Suite Teardown
 
 *** Test Cases ***
-Verify A Card Can Be Created
+erify A Card Can Be Created
```
Comment (mefemefe) [inline tests/trello/gui/trello_gui_cards.robot:12]:
```suggestion
Verify A Card Can Be Created
```

**Review by mefemefe — CHANGES_REQUESTED** [review]

**Review by mefemefe — APPROVED** [review]

**Comment by nicolasvallejo13** [comment]
.

**Comment by nicolasvallejo13** [comment]
solved lint problems and working all tests

**Comment by nicolasvallejo13** [comment]
test case to move cards to anohter board

**Comment by nicolasvallejo13** [comment]
test case to move cards to anohter board working and solved lint problems

**Pull request merged**